lib/Parse/Parser.cpp

```
@@ -374,13 +374,16 @@ std::unique_ptr<SourceUnit> Parser::parse() {
       }
       default:
         ConsumeAnyToken();
+        // TODO : fatalParserError
```

Please raise an fatal error.